### PR TITLE
Switch some images to picture tags

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | We want to use picture tags everywhere we have some images and these parts were missing
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Images on every part should work as expected
| Possible impacts? | Images display

### CODE REVIEW

File: templates/checkout/_partials/order-confirmation-table.tpl [hunk 1]
```diff
@@ -11,9 +11,70 @@
...
+
+              <img
+                class="img-fluid"
+                srcset="
+                  {$product.default_image.bySize.default_xs.url},
+                  {$product.default_image.bySize.default_m.url} 2x"
+                loading="lazy"
+                width="{$product.default_image.bySize.default_xs.width}"
+                height="{$product.default_image.bySize.default_xs.height}"
+                {if !empty($product.default_image.legend)}
```
Comment (Hlavtox) [inline templates/checkout/_partials/order-confirmation-table.tpl:41]: This is handled by the core, there will always be a legend. 👍 

`'legend' => !empty($object->meta_title) ? $object->meta_title : $object->name,`

**Review by Hlavtox — APPROVED** [review]
Just check the legends, good otherwise!

**Review by SharakPL — CHANGES_REQUESTED** [review]
Some indentations + legend @Hlavtox mentioned.
Also maybe we could limit the size of the tags eg. put `type` next to `source`:
```
<source type="image/avif"
  srcset="..."
>
```

**Comment by NeOMakinG** [comment]
Thanks a lot for the review @SharakPL @Hlavtox , I'll fix that tomorrow

**Review by Hlavtox — APPROVED** [review]

**Comment by Hlavtox** [comment]
@SharakPL Merge it when you check it ;-)

**Review by SharakPL — APPROVED** [review]

File: modules/productcomments/views/templates/hook/post-comment-modal.tpl [hunk 2]
```diff
@@ -29,10 +29,67 @@
...
+                        <img
+                          class="js-qv-product-cover rounded"
+                          srcset="
+                            {$product.cover.bySize.default_xs.url},
+                            {$product.cover.bySize.default_m.url} 2x"
+                          loading="lazy"
+                          width="{$product.cover.bySize.default_xs.width}"
+                          height="{$product.cover.bySize.default_xs.height}"
+                          alt="{$product.cover.legend}"
+                          title="{$product.cover.legend}"
```
Comment (Hlavtox) [inline modules/productcomments/views/templates/hook/post-comment-modal.tpl:61]: @NeOMakinG maybe > missing here? :D

**Pull request merged**